src/Components/Profile/Header/PlayerStats.jsx

```diff
@@ -27,15 +27,15 @@ class PlayerStats extends React.Component {
                 <div className='stats'>
                     <div className='stat'>
                         <div>wins</div>
-                        {this.props.profileData.wins}
+                        {this.props.profileData.wins ? this.props.profileData.wins : 0}
                     </div>
                     <div className='stat'>
                         <div>losses</div>
-                        {this.props.profileData.losses}
+                        {this.props.profileData.losses ? this.props.profileData.wins : 0}
```

Comment:
Should this not be `this.props.profileData.losses`?

Reply:
hold up we deprecated this way of doing things like 2 months ago what am i doing